public/pages/workflow_detail/workflow_inputs/workflow_inputs.tsx

```diff
@@ -269,6 +285,119 @@ export function WorkflowInputs(props: WorkflowInputsProps) {
     setSearchProvisioned(hasProvisionedSearchResources(props.workflow));
   }, [props.workflow]);
 
+  // Handle console expansion state
+  useEffect(() => {
+    if (props.setIsConsoleExpanded !== undefined) {
+      props.setIsConsoleExpanded(consoleExpanded);
+    }
+  }, [consoleExpanded, props.setIsConsoleExpanded]);
+
+  useEffect(() => {
+    if (props.isConsoleExpanded !== undefined) {
+      setConsoleExpanded(props.isConsoleExpanded);
+    }
+  }, [props.isConsoleExpanded]);
+
+  useEffect(() => {
+    if (props.setConsoleContent) {
+      const consoleContent = (
+        <EuiFlexGroup direction="column" gutterSize="s">
```

Comment:
Strongly suggest to move all console content into a single parent component, and child sub-components, similar to the existing `<Tools />` component, where each tab is a child component. You should be able to reuse most/all of the logic in the existing [Errors](https://github.com/opensearch-project/dashboards-flow-framework/blob/main/public/pages/workflow_detail/tools/errors/errors.tsx) and [ingest](https://github.com/opensearch-project/dashboards-flow-framework/blob/main/public/pages/workflow_detail/tools/ingest/ingest.tsx) components for rendering the content within the console.